README.md

```diff
@@ -1 +1,93 @@
-# microverse-portfolio
\ No newline at end of file
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
```

Comment:
- [ ] As requested by the previous Code reviewer  you should  include the Table of content from the Microvere README Template in your README. Also please link each section with the Table of Content so that when a user clicks on a particular section in the table of content it takes it to that section. It will allow user to easly navigate through you document :+1: 

You can do that like this :+1: 

```bash
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - ...
```

Look at it [here](https://github.com/microverseinc/readme-template/blob/master/README.md)